jconstraints-core/src/main/java/gov/nasa/jpf/constraints/smtlibUtility/smtconverter/SMTLibExportVisitor.java

```diff
@@ -148,7 +176,11 @@ private boolean bvType(Type<?> t) {
 
   @Override
   public Void visit(RegExBooleanExpression n, Void v) {
-    ctx.open("str.in.re");
+    String operator = "str.in_re";
+    if (config.isZ3Mode) {
+      operator = "str.in.re";
+    }
```

Comment:
Perhaps express as ternary?
```suggestion
    String operator = config.isZ3Mode ? "str.in.re" : "str.in_re";
```